### DISCUSSION TITLE
Device config page to scale with display

### DISCUSSION THREAD

**Discussion by ITensEI** [home-assistant/frontend#13247] [Configuration]
Currently the Device page has 3 columns. The container holding them is set to a static width of 1000px. 
The issue is that some sensors have long values which spill over into the sensor name column.
I went into the console to play around with sizing and found that setting the `.container { max-width: 80% }` and the 1st and 3rd column to `width: 20%` makes it a bit nicer to view the long states, but that doesn't stop the text to spill over into the sensor name field. 

![image](https://user-images.githubusercontent.com/9561099/180042143-d299669d-507c-492a-9b57-ae0b936f819c.png)

I tried adjusting the dimensions of the elements and found that setting the sensor state div to 
![image](https://user-images.githubusercontent.com/9561099/180044102-269abefe-71f8-4bb9-94ae-b166f7150403.png)

Makes the text not reach as far left to cover the sensor name
![image](https://user-images.githubusercontent.com/9561099/180044484-e449b17f-1584-49fa-aba6-98d973a20123.png)

Now this is me playing around and I have very little idea of what I'm doing. 
I would love to see this change and maybe have the Logbook be a little longer.

**Comment by nicopasla** [comment]
This is still happening and we should have the option to change the width of the sensor page, cant read properaly 90% of the sensors text
